strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-composer-layout-provider/src/test/java/org/carlspring/strongbox/artifact/coordinates/ComposerArtifactCoordinatesTest.java

```diff
@@ -0,0 +1,96 @@
+package org.carlspring.strongbox.artifact.coordinates;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import edu.emory.mathcs.backport.java.util.Arrays;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.data.jdbc.DataJdbcTest;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/**
+ * Tests for ComposerArtifactCoordinates class
+ *
+ * @author jcoelho
+ */
+public class ComposerArtifactCoordinatesTest
+{
+
+    private static List<String> packages;
+
+    static
+    {
+        packages = new ArrayList<>();
+    }
+
+    @BeforeAll
+    static void setup()
+    {
+
+
+        packages.add("symfony/polyfill-mbstring");
+
+        packages.add("psr/log");
+
+        packages.add("symfony/polyfill-ctype");
+
+        packages.add("guzzlehttp/psr7");
+
+    }
+
+    @Test
+    public void testCreateArtifactFromParse()
+    {
+
+        for (String _package : packages)
```

Comment:
Cosmetic: Could you please change the variable's name to `package`, or `p`?

Reply:
Done!